pages/task/schema/confirm.js

```diff
@@ -14,4 +25,18 @@ module.exports = (task, chosenStatus) => {
       }]
     }
   };
+
+  const isAmendment = get(task, 'type') === 'amendment';
+
+  if (awerbRequired(task)) {
+    schema = {
+      awerb: {
+        ...getAwerbQuestion(isAmendment),
+        automapReveals: true
```

Comment:
Is there a reason we want this here, but not in the other instances of the same thing?

Reply:
This is a new feature @joefitter added that means you don't need to manually map the reveals, e.g. https://github.com/UKHomeOffice/asl-pages/blob/master/pages/project-version/update/submit/views/index.jsx#L6-L26

Reply:
How come it is _only_ used here and not in the other places we ask the question though?